packages/mui-material-next/src/Snackbar/Snackbar.tsx

```diff
@@ -124,26 +139,19 @@ const Snackbar = React.forwardRef(function Snackbar(inProps, ref) {
     getSlotProps: getRootProps,
     externalForwardedProps: other,
     ownerState,
-    additionalProps: {
-      ref,
-    },
+    additionalProps: { ref },
     className: [classes.root, className],
```

Comment:
```suggestion
    className: [classes.root],
```

We don't need to manually destructure this `className` from props and pass it here anymore, it's ok to leave it `other` and let it get passed to `externalForwardedProps` if you are not doing anything else with it, see https://github.com/mui/material-ui/issues/39601 for background!